### PR TITLE
whirlpool script

### PR DESCRIPTION
**Pull request by IAmAllOfYou**
this is the scrpt for whirlpool. it sucks object to another object the possesses the script. right now it is set to suck ONLY the player to that object

### CODE REVIEW

File: Assets/Scripts/Environment behavior/VortexPull.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    
+    void OnTriggerEnter(Collider col)
+     {
+         //need to put code to not pull on walls
+         //Debug.Log("In field");
+         if(col.gameObject.tag == "Player")
+         {
+             objectsInVortex.Add(col.gameObject);
+         }
+
```
Comment (stellarxo) [inline Assets/Scripts/Environment behavior/VortexPull.cs]: You could also check tag for "Fish" to suck in fish

File: Assets/Scripts/Environment behavior/VortexPull.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    [Tooltip("Increases the pull to the center of the whirlpool")]
+    public float strengthOfAttraction;
+    [Tooltip("Increases the roation of elements inside the whirlpool")]
+    public float strengthOfRotation;
+    private List<GameObject> objectsInVortex = new List<GameObject>();
+	// Use this for initialization
+	void Start () 
+    {
+	
+	}
```
Comment (stellarxo) [inline Assets/Scripts/Environment behavior/VortexPull.cs]: Remove this empty function??

File: Assets/Scripts/Environment behavior/VortexPull.cs [hunk 3]
```diff
@@ -0,0 +1,53 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+
+public class VortexPull : MonoBehaviour {
+    [Tooltip("Increases the pull to the center of the whirlpool")]
+    public float strengthOfAttraction;
+    [Tooltip("Increases the roation of elements inside the whirlpool")]
```
Comment (stellarxo) [inline Assets/Scripts/Environment behavior/VortexPull.cs]: "rotation"

**Comment by alisterdev** [comment]
Tested, Vortex Indeed absorbs the player like a freaking bermuda triangle

**Pull request merged**